docs/using/syntax.md

```diff
@@ -763,9 +780,56 @@ a_{21}& =b_{21}&
 We hope to implement this in a future update (see [executablebooks/MyST-Parser#202](https://github.com/executablebooks/MyST-Parser/issues/202))!
 :::
 
+### Math in other block elements
+
+Math will also work when nested in other block elements, like lists or quotes:
+
+```md
+- $$ a = 1 $$
+- \begin{gather*}
+  a_1=b_1+c_1\\a_2=b_2+c_2-d_2+e_2
+  \end{gather*}
+
+> $$ a = 1 $$
+> \begin{gather*}
+  a_1=b_1+c_1\\a_2=b_2+c_2-d_2+e_2
+  \end{gather*}
+```
+
+- $$ a = 1 $$
+- \begin{gather*}
+  a_1=b_1+c_1\\a_2=b_2+c_2-d_2+e_2
+  \end{gather*}
+
+> $$ a = 1 $$
+> \begin{gather*}
+  a_1=b_1+c_1\\a_2=b_2+c_2-d_2+e_2
+  \end{gather*}
+
+### Mathjax and math parsing
+
+When building HTML using the [sphinx.ext.mathjax](https://www.sphinx-doc.org/en/master/usage/extensions/math.html#module-sphinx.ext.mathjax) extension (enabled by default), its default configuration is to also search for `$` delimiters and LaTeX environments (see [the tex2jax preprocessor](https://docs.mathjax.org/en/v2.7-latest/options/preprocessors/tex2jax.html#configure-tex2jax)).
+
+Since such parsing is already covered by the plugins above, MyST-Parser disables this behaviour by overriding the `mathjax_config` option with:
+
+```python
+mathjax_config = {
+  "tex2jax": {
+  "inlineMath": [["\\(", "\\)"]],
+  "displayMath": [["\\[", "\\]"]],
+  "processRefs": False,
+  "processEnvironments": False,
+  }
+}
+```
+
+Since these delimiters are how `sphinx.ext.mathjax` wraps the math content in the built HTML documents.
+
+To inhibit this override, set `override_mathjax=False`.
```

Comment:
is there some way for people to provide their own *extra* mathjax overrides? several folks have asked about this in JB

Reply:
Just use `mathjax_config`? Then now, they will also have to ensure `myst_override_mathjax=False`